data/operators/amenity/vending_machine.json

```diff
@@ -29,6 +29,67 @@
         "operator:wikidata": "Q60790331",
         "vending": "parking_tickets"
       }
+    },
+    {
+      "displayName": "Mera-Serwis SGL (ZTM Warszawa)",
+      "locationSet": {"include": ["pl"]},
+      "tags": {
+        "amenity": "vending_machine",
+        "vending": "public_transport_tickets",
+        "operator": "Mera-Serwis SGL",
+        "network": "network=ZTM Warszawa",
+        "network:wikidata": "Q8066998",
+        "network:wikipedia": "pl:Zarząd Transportu Miejskiego w Warszawie",
+        "currency:PLN": "yes",
+        "payment:coins": "yes",
```

Comment:
I'm troubled with payment related tag

See https://github.com/osmlab/name-suggestion-index/wiki/Judge-Case

Some times we allow the addition of payment, but more time we reject it. @bhousel 

---
In my opinion, NSI shouldn't be a treasure chest with every kind of detailed tags, those should be added after on ground survey. But if this entry belongs to a public transport, we have enough reason believe that every single machine support same payment.

（> <）！

Reply:
In Warsaw, single-operator vending machines accept the same forms of payment.